common/browser_context.go

```diff
@@ -63,9 +66,53 @@ func NewBrowserContext(
 	return &b
 }
 
-// AddCookies is not implemented.
+// AddCookies adds cookies into this browser context.
+// All pages within this context will have these cookies installed.
 func (b *BrowserContext) AddCookies(cookies goja.Value) {
-	k6ext.Panic(b.ctx, "BrowserContext.addCookies(cookies) has not been implemented yet")
+	b.logger.Debugf("BrowserContext:AddCookies", "bctxid:%v", b.id)
```

Comment:
This is optional for tackling right now. You might want to take a look if you have the time 🙇 

Instead of panicking in multiple places, it would be better to add another method called `addCookies`, return errors from it, call it from `AddCookies`, and panic only once in `AddCookies`. And we can eliminate prefixing the panic error only once with the following: `"adding cookies: "` in `AddCookies`.